emcc

```diff
@@ -1257,9 +1270,15 @@ try:
 
   log_time('post-link')
 
+  # Prepare JS libraries for Emscripten linking, extracting and merging .jso
```

Comment:
This part feels wrong to me. We have already processed these `.a` files and extracted their contents, haven't we? We should be able to just move the extracted files from object files to js library files, if they are `.jso`s.